docs/UserGuide.md

```diff
@@ -163,6 +163,9 @@ Fig 2. Inventory & Statistics tab
 
 * The format of all phone parameters is a series of up to 20 digits with no spacing between digits.
 
+* Address fields allow non-empty inputs(inputs that contain non-whitespace characters) with unlimited length and
+  no restriction on the type of characters.
```

Comment:
This seems like it is written for programmers to understand and not the end users. Can you rephrase this so that as a user i understand it easily?